experiments/integrated/ozark/ozark.jl

```diff
@@ -229,42 +229,48 @@ sol = SciMLBase.solve(
 daily = sol.t ./ 3600 ./ 24
 savedir = joinpath(climalsm_dir, "experiments/integrated/ozark/")
 
+# This function will be used to compute averages over diurnal cycles. Input a
+# data series to average over and the total number of days in the data series, 
+# and it will return a vector of the average value at each timestamp in the 
+# day averaged over every day in the series.
+function diurnal_avg(series, days)
+    daily = floor(Int, length(series) / days)
+    daily_data = [
+        [sample for sample in series[i:1:(i + daily - 1)]] for
+        i in 1:daily:(daily * days)
+    ]
+    daily_avgs = [mean([daily_data[i][j] for i in 1:days]) for j in 1:daily]
+    return daily_avgs
+end
+
 # GPP
 model_GPP = [
     parent(sv.saveval[k].canopy.photosynthesis.GPP)[1] for
     k in 1:length(sv.saveval)
 ]
 
-plt1 = Plots.plot(size = (1500, 400))
+GPP_daily_avg_model = diurnal_avg(model_GPP, 110)
```

Comment:
are these values (110, 120, 48) hardcoded for the dataset or will they change if we change t0, tf, dt?

Reply:
They will change if we change the number of days of the simulation, if we change df, or if the datasets have different timesteps. For example in the Ozark dataset the data is in 30 minutes intervals but for Harvard forest they are hourly so the slice you take is different. We can definitely abstract these to variables if we want to

Reply:
thanks for the explanation! I think Ndays is specified in the ozark_simulation file (but subtract the number of days in spinup from this so we dont include that). And maybe we can define the interval for the data in the met_drivers file? I think 1800 is hardcoded elsewhere. 

(feel free to tackle now or we can take care of in a followup PR)